server/controllers/nlp.js

```diff
@@ -17,19 +21,176 @@ let functions = {
                 type: 'histogram'
             }]
         ))
-    }
+    },
+
+    plotLineChartOfColumn: function (dataset, parameters, callback) {
+        let column = parameters[0];
+
+        // SELECT <column> FROM <dataset>
+        bookshelf.Model.extend({tableName: dataset}).fetchAll({columns: [column]}).then(data => callback(
+            [{
+                // map [{<columnName>: <columnValue>}, ...] to [<columnValue>, ...]
+                x: data.map((value) => value.attributes[column]),
+                type: 'scatter'
+            }]
+        ))
+    },
+    plotScatterOfTwoColumns: function (dataset, parameters, callback) {
+        let column1 = parameters[0];
+        let column2 = parameters[1];
+
+        // SELECT <column> FROM <dataset>
```

Comment:
copy past error, or?

Reply:
Do you mean that it should be <column1, column2> in the comment?

Reply:
`// SELECT <column1>, <column2> FROM <dataset>`